### DISCUSSION TITLE
RF over fiber

### DISCUSSION THREAD

**Discussion by chaloz** [Time-Appliances-Project/Time-Card#112] [Q&A]
Hi,

I was just looking at possible options for GNSS over Fibre, and lo and behold, [here](https://github.com/opencomputeproject/Time-Appliance-Project/tree/master/Incubation/Hardware/SFP_RFoF) is this Incubation board from @julianstj1. I didn't think it could be that simple.

Can you, @julianstj1, give a little brief about how this actually works? I would have thought that the SFP's module comparator would not be as sensitive to be able to transfer GNSS spectrum. I was thinking that the power levels are so low, that there would be a constant zero after the SFP's module comparator does its job.

Thanks for any hints!

**Comment by julianstj1** [comment]
@chaloz  this is a first prototype board I designed to test GNSS over fiber and more general RF over fiber. I worked with a company (can't publicly disclose at the moment as far as I know) to design and build true "linear" SFP modules, that have a mostly flat frequency transfer curve above multiple GHz. So this board is intended as the host board to debug and bring-up these modules, and test them with different RF bands. That's why there's configurable attenuators and multiple paths for different SFP ports, intended to test WiFi over fiber or GNSS over fiber for example.

If you want more details than I can publicly discuss on github, please send me an email (julianstj@meta.com) , and let me know if you want to collaborate or are also interested in this hardware development. I can also check with the SFP vendor and possibly connect you with them too if you're interested.

**Comment by julianstj1** [comment]
Added some documentation about SFP RFoF design in the Incubation folder.
